pages/app-layout/multi-layout-iframe.page.tsx

```diff
@@ -17,6 +17,7 @@ import * as toolsContent from './utils/tools-content';
 function InnerApp() {
   return (
     <AppLayout
+      {...{ __disableRuntimeDrawers: true }}
```

Comment:
Not relevant to the matter of the fix, just suppress warnings in browser console, which distract from debugging